pkg/build/api/register.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 const GroupName = ""
+const FutureGroupName = "build.openshift.io"
```

Comment:
> builds? To match "apps" and "extensions"?

"batch" and "imagepolicy"?  We've gone back and forth before in one of @mfojtik's pulls.  I kind of expect that a group is a singular construct.  Probably because I say, "`<collective noun>` **is**", not  "`<collective noun>` **are**"